make fmt fails due to missing buildifier

following steps in devel/README.md, step to `make fmt` fails:
```
$ make fmt
bin/fmt.sh: line 26: buildifier: command not found
Makefile:28: recipe for target 'fmt' failed
make: *** [fmt] Error 127
```
The `buildifier` dependency is missing from the Bazel instructions.

--- a/devel/README.md
+++ b/devel/README.md
@@ -75,6 +75,8 @@ export PATH=$PATH:$GOPATH/bin
 Istio components are built using the Bazel build system. See
 [here](https://bazel.build/versions/master/docs/install.html) for the
 installation procedures.
+In addition to Bazel itself, you should install the Bazel buildifier tool from
+[here](https://github.com/bazelbuild/buildtools).
 
 ### Setting up Docker
 

